velox/docs/configs.rst

```diff
@@ -887,6 +887,13 @@ Spark-specific Configuration
        Joda date formatter performs strict checking of its input and uses different pattern string.
        For example, the 2015-07-22 10:00:00 timestamp cannot be parsed if pattern is yyyy-MM-dd because the parser does not consume whole input.
        Another example is that the 'W' pattern, which means week in month, is not supported. For more differences, see :issue:`10354`.
+   * - spark.legacy_statistical_aggregate
```

Comment:
Ditto. This configuration seems unnecessary for this PR. If we would change Presto's implementation and make the behaviour configurable, we may need it then.

Reply:
It is configurable in Spark, default false